src/components/Form/CreateProfileForm/DiscountData/DiscountData.tsx

```diff
@@ -156,6 +156,10 @@ const DiscountData = ({
           setInitialValues({
             discounts: discounts.items.map((discount: Discount) => ({
               ...discount,
+              discountUrl:
+                discount.discountUrl === null
+                  ? undefined
+                  : discount.discountUrl,
```

Comment:
fromNullable(discount.discountUrl).toUndefined() ❤️

Reply:
Since we are here...[055b49a](https://github.com/pagopa/cgn-onboarding-portal-frontend/pull/171/commits/055b49a8a6c508bdfd33f160e0e60a1cd1dcfb27)